ckanext/data_qld/templates/package/resource_read.html

```diff
@@ -110,11 +110,20 @@ <h2>{{ _('Data Dictionary') }}</h2>
 {% endblock %}
 
 {% block resource_read_title %}
-    <h1 class="page-heading">{{ h.resource_display_name(res)}}
-    {% if res.schema %}
-        {{ h.get_validation_badge(res)|safe }}
-        {% set type = 'asset' %}
-        {% include 'validation/snippets/validation_style_' ~ type ~ '.html' %}
-    {% endif %}
+    <h1 class="page-heading">
+        {% if res.qld_hidden %}
+            <span class="dataset-private label label-inverse">
+                <i class="fa fa-lock"></i>
+                {{ _("Hidden" )}}
+            </span>
+        {% endif %}
+
+        {{ h.resource_display_name(res)}}
+
+        {% if res.schema %}
+            {{ h.get_validation_badge(res)|safe }}
+            {% set type = 'asset' if h.is_ckan_29() else 'resource' %}
```

Comment:
CKAN 2.8 support has been dropped, this helper is going away.

Reply:
Fixed. I've preserved the style from your repo.